src/components/data-exploration/differential-expression-tool/DiffExprCompute.jsx

```diff
@@ -14,14 +14,20 @@ import { loadCellSets } from 'redux/actions/cellSets';
 import { setComparisonGroup, setComparisonType } from 'redux/actions/differentialExpression';
 import { getCellSets } from 'redux/selectors';
 import { composeTree } from 'utils/cellSets';
+import checkCanRunDiffExpr from 'utils/differentialExpression/checkCanRunDiffExpr';
 
 const { Option, OptGroup } = Select;
 
 const ComparisonType = Object.freeze({ BETWEEN: 'between', WITHIN: 'within' });
 const getCellSetKey = (name) => (name?.split('/')[1] || name);
 const getRootKey = (name) => name?.split('/')[0];
 
-const MIN_NUM_CELLS_IN_GROUP = 10;
+const canRunDiffExprResults = {
+  TRUE: 'TRUE',
+  FALSE: 'FALSE',
+  INSUFFICIENT_CELLS_WARNING: 'INSUFFICIENT_CELLS_WARNING',
+  INSUFFCIENT_CELLS_ERROR: 'INSUFFCIENT_CELLS_ERROR'
+}
```

Comment:
We could move this to checkCanRunDiffExpr

Reply:
moved